modules/integration_azure-functions/conf/01-function-errors.yaml

```diff
@@ -0,0 +1,16 @@
+
+module: functions
+name: "Azure Function errors"
+id: errors
+signals:
+  signal:
+    metric: azure.function.errors
+    extrapolation: 0
+rules:
+  critical:
+    threshold: 1
+    comparator: ">="
+filtering: "filter('is_Azure_Function', 'true')"
+aggregation: ".mean(by=['azure_function_name', 'subscription_id']"
+disabled: true
```

Comment:
in fact it is not bad to keep this enabled by default.

if there is not any instrumented function using the wrapper so this detector will never get data so it is safe.

while you already inform the user of this dependency it seems fine for me.

Reply:
Key removed, now the detector is activated by default